### DISCUSSION TITLE
I want to use unmodified hotkeys.

### DISCUSSION THREAD

**Discussion by marvelph** [mikekazakov/nimble-commander#177] [Q&A] [answered]
I want to assign an unmodified hotkey to “Copy To..." or something similar, but it beeps and I can't seem to set it.
On the other hand, it seems that unqualified hotkeys can be assigned to “Show Preview" and other hotkeys.
I want to be able to perform most file operations without using modifier keys.

**Comment by mikekazakov** [comment, marked as answer]
That's by the MacOS UX - the majority of the actions are driven via menu items, which have their limitations for what a hotkey can be.  
Partially discussed here: https://github.com/mikekazakov/nimble-commander/discussions/151  
This limitation can be circumvented by introducing a custom system for assigning secondary hotkeys without these limitation, but such system is simply not implemented yet.

**Reply by marvelph** [reply]
Thanks for the response.
We look forward to future expansions.
